js/package.json

```diff
@@ -97,5 +85,28 @@
     "ts-jest": "^26.5.4",
     "typescript": "^4.2.4",
     "yarn-deduplicate": "^3.1.0"
+  },
+  "resolutions": {
+		"**/@types/react": "^17.0.0"
+  },
+  "sideEffects": [
+    "style/**/*.css",
+    "style/index.js"
+  ],
+  "styleModule": "style/index.js",
```

Comment:
is this necessary? don't we bundle the css assets

Reply:
I remember @jasongrout explaining at one point about the need for css-in-js modules. But I don't remember why. Since this is the same pattern used in all of the core pkgs as well as most of the larger 3rd party extensions, I am going to keep this in for now

Reply:
just want to make sure theres not 2 copies of the CSS assets, should be easy to check on the page load

Reply:
i'll double check that

Reply:
That styleModule thing is to prevent duplication of CSS from including CSS from dependencies. For many more technical details, see https://github.com/jupyterlab/jupyterlab/pull/9407.

Reply:
Just checked, in terms of the stylesheets loaded onto the page, our css resources are not getting copied/doubled.

I also asked about the css-in-js pattern on gitter. I'll see if anyone can illuminate this a little further

Reply:
ah ok makes sense, extensions that rely on common web pack need this to "register" their css assets. I'm satisfied (though this should probably be more clear on the extension docs)